service/tbc/crawler.go

```diff
@@ -439,6 +489,7 @@ func (s *Server) SyncIndexersToHeight(ctx context.Context, height uint64) error
 			return fmt.Errorf("tx indexer: %w", err)
 		}
 	}
+	log.Infof("done working")
```

Comment:
Was this placed here for debugging? Can it be removed or changed to a debug log?